anet-dictionary.yml

```diff
@@ -809,6 +809,156 @@ fields:
         - endOfTourDate
         - biography
         - arrayFieldName
+      assessments:
+        - recurrence: once
+          relatedObjectType: report
+          questions:
+            role:
+              test: $.subject.position.organization[?(@property === "shortName" && @.match(/^LNG/i))]
+              type: enum
+              label: Role of Interpreter
+              choices:
+                Translating:
+                  label: Translating
+                Advising:
+                  label: Advising
+            Overall Meeting Evaluation:
```

Comment:
I assume the property names correspond with the property names which will be used to capture the assesment in the database. I think it better to prevent spaces, to eae the use of expressions like the `visibleWhen`.

I suggest to use `overall-meeting-evaluation`.

Reply:
Better to use camelCase for YAML/JSON object properties, so `overallMeetingEvaluation` etc.